src/js/itemsCounter.test.js

```diff
@@ -0,0 +1,12 @@
+import { itemsCounter } from './itemsCounter';
+
+test('items counter check ', () => {
```

Comment:
- [ ] It is good practice to test your counter for possible errors. (What happens if the array is empty, What happens if I pass a string instead of an array, etc...)